configs/containers/docker-ubuntu-clang-mpich.yaml

```diff
@@ -110,7 +109,7 @@ spack:
       os: ubuntu:20.04
       spack:
         url: https://github.com/jcsda/spack
-        ref: spack-stack-1.5.0
+        ref: jcsda_emc_spack_stack
```

Comment:
An actual bug fix, this should not have been updated when `spack-stack-1.5.0` was merged into develop.